blog/electron-20.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Electron 20.0.0
+date: 2022-08-02:00:00.000Z
+authors:
+    - name: ckerr
+      url: 'https://github.com/ckerr'
+      image_url: 'https://github.com/ckerr.png?size=96'
+slug: electron-20-0
+
+---
+
+Electron 20.0.0 has been released! It includes upgrades to Chromium `104`, V8 `10.4`, and Node.js `16.15.0`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 20.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://www.electronjs.org/releases/stable). Continue reading for details about this release and please share any feedback you have!
+
+## Notable Changes
+
+### New Features
+
+* Added immersive dark mode on Windows. [#34549](https://github.com/electron/electron/pull/34549)
+* Added support for panel-like behavior. Window can float over full-screened apps. [#34665](https://github.com/electron/electron/pull/34665)
+* Updated Windows Control Overlay buttons to look and feel more native on Windows 11. [#34888](https://github.com/electron/electron/pull/34888)
+
```

Comment:
Maybe we could add the two most major breaking changes here, just so they're very visible for users who are scanning the blog post?

```suggestion
* Renderers are now sandboxed by default unless `nodeIntegration: true` or `sandbox: false` is specified. [#35125](https://github.com/electron/electron/pull/35125)
* Added safeguards when building native modules with nan. [#35160](https://github.com/electron/electron/pull/35160)

```

Reply:
+1